### PR TITLE
feat: Add schema export

### LINKED ISSUE #45: Migrate Schema
Is there any way to save current schema of classes (with all the fields) and to import it later?

### PR DESCRIPTION
**Pull request by dblythy**
### New Pull Request Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am creating this PR in reference to an [issue](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).

### Issue Description

Currently, it is not possible to export SCHEMA. The DefinedSchema guide recommends doing it via JS, which is not ideal.

https://docs.parseplatform.org/defined-schema/guide/#2-extract-schemas-using-js-console

Closes: #45

### Approach

Adds ability to extract SCHEMAS.

Screenshot 2023-01-20 at 1 59 37 pm
Screenshot 2023-01-20 at 2 00 11 pm

### TODOs before merging

- [ ] Add tests
- [ ] Add changes to documentation (guides, repository pages, in-code descriptions)

### CODE REVIEW

File: src/dashboard/Data/Browser/BrowserToolbar.react.js [hunk 1]
```diff
@@ -257,6 +258,10 @@ let BrowserToolbar = ({
             text={'Export all rows'}
             onClick={() => onExportSelectedRows({ '*': true })}
           />
+          <MenuItem
+            text={'Export SCHEMA'}
```
Comment (mtrezza) [inline src/dashboard/Data/Browser/BrowserToolbar.react.js]:
```suggestion
            text={'Export schema'}
```

**Review by mtrezza — CHANGES_REQUESTED** [review]

**Review by mtrezza — APPROVED** [review]
Looks good!

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.1.0-alpha.3](https://github.com/parse-community/parse-dashboard/releases/tag/5.1.0-alpha.3)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.1.0-beta.2](https://github.com/parse-community/parse-dashboard/releases/tag/5.1.0-beta.2)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.1.0](https://github.com/parse-community/parse-dashboard/releases/tag/5.1.0)

**Pull request merged**